src/apps/service/apps/jobs/submissions.clj

```diff
@@ -141,7 +141,7 @@
   [user submission]
   (let [output-dir (ft/build-result-folder-path submission)]
     (try+
-     (data-info/get-file-stats user [output-dir])
+     (data-info/get-path-info user :paths [output-dir] :filter-include "path")
```

Comment:
It looks like this function only returns the `output-dir` path it's checking with `stat-gatherer`. This is probably some old code, because it seems like it could be refactored to use `existence-checker` instead (but that's not necessary for this PR).